### PR TITLE
tell user a topic is not ready when it is not ready

### PR DESCRIPTION
**Pull request by justindowning**

### CODE REVIEW

File: commands/topics_info.js [hunk 1]
```diff
@@ -72,13 +72,16 @@ function * kafkaTopic (context, heroku) {
     let forTopic = info.topics.filter((t) => t.name === topic)
```
Comment (uhoh-itsmaciek) [inline commands/topics_info.js]: As long as you're here, do you mind changing this `filter` to a `find`? Then you can change `(forTopic.length === 0)` below to `(!forTopic)` and avoid indexing into `forTopic` and just use it directly in the other references below.

Comment (justindowning) [inline commands/topics_info.js]: 👍

File: commands/topics_info.js [hunk 2]
```diff
@@ -72,13 +72,16 @@ function * kafkaTopic (context, heroku) {
...
 
     if (forTopic.length === 0) {
-      cli.error(`topic not found ${topic}`)
-      cli.exit(1)
+      cli.exit(1, `topic not found ${topic}`)
```
Comment (uhoh-itsmaciek) [inline commands/topics_info.js]: This would read better as `topic ${topic} not found`, no?

**Comment by uhoh-itsmaciek** [comment]
A couple of minor comments not strictly related to your change. If you can update, that's great, if not I can pick that up later and +1

**Pull request merged**